source/components/organisms/Step/StepFooter/StepFooter.tsx

```diff
@@ -203,6 +207,7 @@ StepFooter.propTypes = {
     start: PropTypes.func,
     /** action to return to main form */
     goToMainForm: PropTypes.func,
+    goToMainFormAndNext: PropTypes.func,
     /** whether we are at the last of the main steps */
```

Comment:
hmmm, döp gärna parametern till det du försöker förtydliga med en kommentar

Reply:
Can we maybe keep the comments to things that are part of the PR? It's good to want to improve the overall code, but we have a deadline, so a bit of focus is probably a good idea right now...

Reply:
Fixed.